packages/client/tests/e2e/driver-adapters-accelerate/prisma/schema.prisma

```diff
@@ -0,0 +1,36 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./db"
+}
```

Comment:
Technically, Accelerate does not support the `sqlite` provider.
So it would be best if we had a special error for that, which I guess we don't have at the moment?

Reply:
Yeah, I am not sure what would happen in this case. I'll change it to `postgresql` so it is cleaner at least.